### PR TITLE
Add -K (--ask-sudo-pass) flag to ansible-pull.

### PR DESCRIPTION
**Pull request by stevetjoa**
Until now, I comfortably used ansible-playbook in my personal workflow with the -K flag. I recently tried ansible-pull but soon realized that it lacks the -K flag and many other flags found in ansible-playbook. This simple commit adds the -K flag to ansible-pull. I tested it on my machine and it appears to work. Without it, I (and other users of the system) would need to run ansible-pull as a NOPASSWD user such as root, something I'd prefer not to do.

Since ansible-pull simply calls ansible-playbook, might it make sense for most, if not all, of the flags available in ansible-playbook to be available in ansible-pull?

### CODE REVIEW

File: bin/ansible-pull [hunk 1]
```diff
@@ -123,6 +123,8 @@ def main(args):
                       default=DEFAULT_REPO_TYPE,
                       help='Module name used to check out repository.  '
                       'Default is %s.' % DEFAULT_REPO_TYPE)
+    parser.add_option('-K', '--ask-sudo-pass', default=False, dest='ask_sudo_pass', action='store_true',
```
Comment (stevetjoa) [inline bin/ansible-pull:126]: copied directly from ansible/lib/utils/**init**.py

**Comment by mpdehaan** [comment]
So I take it you are using ansible-pull interactively?

It was generally intended to be run from cron/etc

Not arguing against the flag, just wanted to be clear that this won't be useful for most uses of ansible-pull, which is why it's not there :)

**Comment by stevetjoa** [comment]
Ah, I see. Correct, I am using it interactively. I might still try to use ansible-pull, even interactively outside cron, because I like the idea of checking into a central repository -- just not as root. However, please feel free to proceed as you wish.

**Comment by mpdehaan** [comment]
Hi! 

Thanks for this.   I noticed from our automatic scanner that this pull request has merge commits in it.  As a general rule, we don't accept merge commits in contributions.  If you would like this in, can you please resubmit this?

A good tip is to always use git pull --rebase vs git pull, or git rebase vs git merge, and always start a new branch for each topic.

(Also, apologies on the delay, we do recognize we have a large backlog).

**Comment by stevetjoa** [comment]
Ah right, sorry about that. Let me see how to fix this, and I will get back to you.

**Comment by stevetjoa** [comment]
Resubmitted as issue #5864. Thanks a lot.

**Pull request closed without merging**